src/Styleguide/Components/__stories__/Tabs.story.tsx

```diff
@@ -90,6 +90,13 @@ storiesOf("Styleguide/Components", module).add("Tabs (Simple)", () => {
           />
         </Tabs>
       </Section>
+      <Section title="Supports empty(null) tabs by ignoring them">
+        <Tabs>
+          {null}
```

Comment:
i.e. if this was `false`... (see above comment)

Reply:
Yea... I guess we should also check for false. Will update in a min.

Theoretically there might be the case where all the tabs are null/false and in this case selected is going to be -1, but I don't think it's a realistic scenario anyways.